source/postprocess/melt_statistics.cc

```diff
@@ -65,14 +66,15 @@ namespace aspect
             // fill material model inputs
             fe_values.reinit (cell);
             in.reinit(fe_values, cell, this->introspection(), this->get_solution());
```

Comment:
This is the place where you need to add `MeltHandler<dim>::create_material_model_outputs(output);`